LastFM/plugin.py

```diff
@@ -151,6 +151,16 @@ def get_artist_tags(self, artist, irc):
 
         return tags
 
+    def check_apiKey(self, irc):
```

Comment:
Why is it named `check_apiKey`? With a name like that It sounds like it will return a boolean, not the api key.